jobserver/migrations/0003_remove_global_projectdeveloper.py

```diff
@@ -0,0 +1,32 @@
+from django.db import migrations
+
+from jobserver.authorization.roles import DeploymentAdministrator, ProjectDeveloper
+
+
+def change_user_role(apps, from_role, to_role):
+    User = apps.get_model("jobserver", "User")
+    for user in User.objects.filter(roles__contains=from_role):
+        user.roles.remove(from_role)
+        user.roles.add(to_role)
```

Comment:
This doesn't work, it needs to be `append` not `add`.

```
  File "/home/lucy/datalab/opensafely-core/job-server/jobserver/migrations/0003_remove_global_projectdeveloper.py", line 10, in change_user_role
    user.roles.add(to_role)
    ^^^^^^^^^^^^^^
AttributeError: 'list' object has no attribute 'add'
```

Reply:
Thanks - is there a reason CI didn't pick this up?

Reply:
Archaeologists, see #4573 ⛏️